employee-rostering-shared/src/main/java/org/optaweb/employeerostering/shared/employee/Employee.java

```diff
@@ -104,7 +111,15 @@ public void setSkillProficiencySet(Set<Skill> skillProficiencySet) {
         this.skillProficiencySet = skillProficiencySet;
     }
 
-    @Override
+    public Contract getContract() {
+        return contract;
+    }
+
+    public void setContract(Contract contract) {
+        this.contract = contract;
+    }
+
+    /*@Override
```

Comment:
Don't comment it out, just delete it directly

Reply:
Thanks for pointing this out; I don't know if I meant to delete it/edit it or I just commented it out for testing. Will try uncommenting.